app/controllers/v1/purls_controller.rb

```diff
@@ -23,6 +23,8 @@ def update
     end
 
     def destroy
+      return render json: { error: "already deleted" }, status: :conflict if @purl.deleted?
```

Comment:
Can you say a little more about using HTTP 409 to respond to a request to delete an already deleted resource? I could see 409s being the best choice, but could also see a case being made for 204 or 404. Is it helpful to consumers to return a 4xx status? Does that signal "do no more, the deed is done?"

Reply:
Yes, I think it's the choice because you can't make a transition from the current state.  It's not an acceptable request.  This is what we use in DSA on similar occurrences: https://github.com/sul-dlss/dor-services-app/blob/fffc87e50b2716f1d36b39876ded9abf834d0cac/openapi.yml#L159-L160

Reply:
Sounds good, thanks